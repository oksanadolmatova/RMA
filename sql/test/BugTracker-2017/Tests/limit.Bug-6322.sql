select
  subq_0.c3 as c2
from 
  (select  
        sample_0.proj4text as c3,
        sample_0.auth_name as c7
      from 
        sys.spatial_ref_sys as sample_0
      where true
      limit 14) as subq_0
where (true)
  or ((select pc from sys.tracelog)
       is not NULL);
