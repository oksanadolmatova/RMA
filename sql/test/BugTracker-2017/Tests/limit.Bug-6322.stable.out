stdout of test 'limit.Bug-6322` in directory 'sql/test/BugTracker-2017` itself:


# 12:52:04 >  
# 12:52:04 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36358" "--set" "mapi_usock=/var/tmp/mtest-19106/.s.monetdb.36358" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 12:52:04 >  

# MonetDB 5 server v11.28.0
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2017', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.581 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://rocks075.scilens.private:36358/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-19106/.s.monetdb.36358
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 12:52:04 >  
# 12:52:04 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-19106" "--port=36358"
# 12:52:04 >  

#select
#  subq_0.c3 as c2
#from 
#  (select  
#        sample_0.proj4text as c3,
#        sample_0.auth_name as c7
#      from 
#        sys.spatial_ref_sys as sample_0
#      where true
#      limit 14) as subq_0
#where (true)
#  or ((select pc from sys.tracelog)
#       is not NULL);
% sys.L15 # table_name
% c2 # name
% varchar # type
% 93 # length
[ "+proj=longlat +ellps=bessel +towgs84=595.48,121.69,515.35,4.115,-2.9383,0.853,-3.408 +no_defs"	]
[ "+proj=longlat +ellps=aust_SA +no_defs"	]
[ "+proj=longlat +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +no_defs"	]
[ "+proj=longlat +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +no_defs"	]
[ "+proj=longlat +ellps=bessel +towgs84=682,-203,480,0,0,0,0 +no_defs"	]
[ "+proj=longlat +ellps=airy +no_defs"	]
[ "+proj=longlat +ellps=mod_airy +no_defs"	]
[ "+proj=longlat +ellps=aust_SA +no_defs"	]
[ "+proj=longlat +ellps=bessel +no_defs"	]
[ "+proj=longlat +a=6377492.018 +b=6356173.508712696 +no_defs"	]
[ "+proj=longlat +ellps=bess_nam +no_defs"	]
[ "+proj=longlat +a=6378293.645208759 +b=6356617.987679838 +no_defs"	]
[ "+proj=longlat +ellps=clrk66 +no_defs"	]
[ "+proj=longlat +a=6378450.047548896 +b=6356826.621488444 +no_defs"	]

# 12:52:04 >  
# 12:52:04 >  "Done."
# 12:52:04 >  

